ui/templates/base.html

```diff
@@ -9,6 +10,7 @@
   <body>
     {% block content %}
     {% endblock %}
+  <script type="text/javascript" src="{{ outer_src }}"></script>
```

Comment:
No that's fine, just want to understand. I have been thinking in terms of subdomains: www and app. Not sure others agree with that.